src/bridge/IBridge.sol

```diff
@@ -13,7 +13,7 @@ interface IBridge {
         bytes32 indexed beforeInboxAcc,
         address inbox,
         uint8 kind,
-        address sender,
+        address indexed sender,
```

Comment:
would prefer to have this in another PR for consideration separately due to how it changes the existing abi